.github/workflows/deploy.yml

```diff
@@ -0,0 +1,157 @@
+name: Test, Build, and Deploy - CI/CD
+
+on:
+  workflow_dispatch:
```

Comment:
good use of these triggers

Reply:
Oops, it shouldn't be there. I only used it for testing.